README.adoc

```diff
@@ -254,7 +244,9 @@ tracing of particular methods. You can also inject a custom `Tracer` object to c
 
 === Enabling distributed tracing without code instrumentation
 
-Because tracing of all JAX-RS methods is enabled by default, you need only to enable [hotspot=mpOpenTracing file=0]`MicroProfile OpenTracing` feature and the [hotspot=zipkinUsr file=0]`Zipkin` user feature in the `server.xml` file to see some basic traces in Zipkin.
+Because tracing of all JAX-RS methods is enabled by default, you need only to enable 
```

Comment:
Because tracing of all JAX-RS methods is enabled by default, you need only to enable 
[hotspot=mpOpenTracing file=0]`MicroProfile OpenTracing` feature and the [hotspot=zipkinUsr file=0]`Zipkin` 
user feature in the `server.xml` file to see some basic traces in Zipkin.
--->
Because tracing is enabled by default for all JAX-RS methods , you need to enable only the
[hotspot=mpOpenTracing file=0]`MicroProfile OpenTracing` feature and the [hotspot=zipkinUsr file=0]`Zipkin` 
user feature in the `server.xml` file to see some basic traces in Zipkin.